src/main/java/org/broadinstitute/hellbender/engine/AssemblyRegionEvaluator.java

```diff
@@ -0,0 +1,21 @@
+package org.broadinstitute.hellbender.engine;
+
+import org.broadinstitute.hellbender.utils.activityprofile.ActivityProfileState;
+
+/**
+ * Classes that implement this interface have the ability to evaluate how likely it is that a site is "active"
+ * (contains potential real variation).
+ */
+@FunctionalInterface
+public interface AssemblyRegionEvaluator {
+
+    /**
+     * Given a pileup, returns an ActivityProfileState containing the probability (0.0 to 1.0) that it's an "active" site.
+     *
+     * @param locusPileup reads pileup to examine
+     * @param referenceContext reference base overlapping the pileup locus
+     * @param featureContext features overlapping the pileup locus
+     * @return probability between 0.0 and 1.0 that the site is active
+     */
+    ActivityProfileState isActive( final AlignmentContext locusPileup, final ReferenceContext referenceContext, final FeatureContext featureContext );
```

Comment:
given that we have a class `ReadPileup` the name of `locusPileup` is confusing a bit